packages/react-components/src/organisms/__tests__/TeamGroupCard.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { createListGroupResponse } from '@asap-hub/fixtures';
+import userEvent from '@testing-library/user-event';
+
+import TeamGroupTools from '../TeamGroupCard';
+
+it('renders team group tools with 1 group', () => {
+  const { getByRole, queryByRole } = render(
+    <TeamGroupTools {...createListGroupResponse(1)} />,
+  );
+  expect(getByRole('heading', { level: 3 }).textContent).toMatchInlineSnapshot(
+    `"Team Groups (1)"`,
+  );
+  expect(queryByRole('button')).not.toBeInTheDocument();
+});
+
+it('renders team group tools with 2 group', () => {
+  const { getByRole, queryByRole } = render(
+    <TeamGroupTools {...createListGroupResponse(2)} />,
+  );
+  expect(getByRole('heading', { level: 3 }).textContent).toMatchInlineSnapshot(
+    `"Team Groups (2)"`,
+  );
+  expect(queryByRole('button')).not.toBeInTheDocument();
+});
+
+it('renders team group tools with 1 group and 1 team', () => {
+  const { getByText } = render(
+    <TeamGroupTools {...createListGroupResponse(1, { teamsCount: 1 })} />,
+  );
+
+  expect(getByText('1 Team')).toBeInTheDocument();
+});
+it('renders team group tools with 1 group and 2 teams', () => {
+  const { getByText } = render(
+    <TeamGroupTools {...createListGroupResponse(1, { teamsCount: 2 })} />,
+  );
+  expect(getByText('2 Teams')).toBeInTheDocument();
+});
+it('renders team group tools with 3 groups', () => {
```

Comment:
Please make the test titles descriptive of what they are checking, this does not explain that it's expecting a show more button to appear as the group count increases to 3